packages/odf/components/s3-browser/upload-objects/UploadSidebar.tsx

```diff
@@ -0,0 +1,174 @@
+  const isComplete = uploadedFiles + failedFiles === totalFiles;
+
+  return (
+    <DrawerPanelContent
+      isResizable
+      defaultSize="500px"
+      minSize="150px"
+      maxSize="unset"
+      height="unset"
+    >
+      <div className="odf-upload-sidebar__drawer-head">
+        <DrawerHead>
+          <Flex direction={{ default: 'column' }}>
+            <FlexItem>
+              <Title headingLevel="h3">{t('Uploads')}</Title>
+            </FlexItem>
+            <FlexItem>
+              <Title headingLevel="h4">
+                <span>
+                  {t('{{uploadedFiles}} of {{totalFiles}} files uploaded', {
+                    uploadedFiles,
+                    totalFiles,
+                  })}
+                  &nbsp;
+                  {isComplete ? (
+                    <ResourceStatus>
+                      <Status status={t('Complete')} title={t('Completed')} />
+                    </ResourceStatus>
+                  ) : (
+                    <ResourceStatus>
+                      <Status status={t('Uploading')} title={t('Ongoing')} />
+                    </ResourceStatus>
+                  )}
+                </span>
+              </Title>
+            </FlexItem>
+            {isComplete ? (
+              <>
+                <FlexItem>
+                  {t('Succeeded: {{uploadedFiles}}', { uploadedFiles })}
+                </FlexItem>
+                <FlexItem>
+                  {t('Failed files: {{failedFiles}}', {
+                    failedFiles: totalFiles - uploadedFiles,
+                  })}
+                </FlexItem>
+                <FlexItem>
+                  {t('Completion time: {{totalTimeElasped}}', {
+                    totalTimeElasped,
+                  })}
+                </FlexItem>
+              </>
+            ) : (
+              <>
+                <FlexItem>
+                  {t('Total Remaining: {{totalReamining}}', { totalReamining })}
+                </FlexItem>
+                <FlexItem>
+                  {t('Estimated time remaining: {{timeRemaining}}', {
+                    timeRemaining,
+                  })}
+                </FlexItem>
+                <FlexItem>
+                  {t('Transfer rate: {{uploadSpeed}}', { uploadSpeed })}
+                </FlexItem>
+              </>
+            )}
+          </Flex>
+          <DrawerActions>
+            <DrawerCloseButton onClick={onClose} />
+          </DrawerActions>
+        </DrawerHead>
+      </div>
+      <DrawerContentBody>
+        <Alert
+          title="How uploading works?"
+          isInline
+          variant={AlertVariant.info}
+          className="pf-v5-u-mb-sm"
+        >
+          <Trans t={t}>
+            Standard uploads have a size limit of up to 5 TB in S3. For large
+            objects, something or the other will happen. We need to improve this
+            alert. Or not show this. Doesnt make sense to repeat your messages
+            in UI.
```

Comment:
:smile:

Reply:
I know microcopy is pending, still good not to show wrong text on UI !
```suggestion
            objects,
```